ambari-funtest/src/test/resources/stacks/HDP/2.0.6.1/services/FLUME/metainfo.xml

```diff
@@ -55,7 +55,10 @@
       </commandScript>
       
       <requiredServices>
-        <service>HDFS</service>
+        <service>
+          <name>HDFS</name>
+          <scope>INSTALL</scope>
```

Comment:
Can you rename "scope" to "type" or "dependencyType"? We might end up adding scope in the future to scope to HOST,CLUSTER etc.